src/Discovery/DiscoveryDetails/DataDownloadList/DataDownloadList.tsx

```diff
@@ -16,6 +17,7 @@ interface DataDownloadListProps {
 }
 
 const DataDownloadList = ({
+  resourceFieldValueIsValid,
```

Comment:
I think you need to use this value to check if you can pass `sourceFieldData` into `ProcessData()` at https://github.com/uc-cdis/data-portal/pull/1476/files#diff-1dbfc504269445ba617c8cf99d89caf9865795dc6db82e0d0fa12b14313a8770R27 below. Because there is no guarantee that the `sourceFieldData` here will be an array, which will cause problems down the road in `ProcessData()` when it tries to process the input as an array

Reply:
That is a good point, the assignment to the const ```data``` has been updated to only process the data if the ```resourceFieldValueIsValid``` is true. 

```
  const data: [] = resourceFieldValueIsValid
    ? ProcessData(sourceFieldData)
    : [];
```